src/main/scala/org/phenoscape/kb/ingest/bgee/BgeeExpressionToOWL.scala

```diff
@@ -0,0 +1,63 @@
+package org.phenoscape.kb.ingest.bgee
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+import java.io._
+import scala.io.Source
+import org.phenoscape.scowl.Functional._
+
+import org.apache.commons.lang3.StringUtils
+import org.phenoscape.kb.ingest.util.Vocab._
+import org.phenoscape.kb.ingest.util.ExpressionUtil
+import org.phenoscape.kb.ingest.util.OBOUtil
+import org.phenoscape.kb.ingest.util.OntUtil
+import org.phenoscape.scowl._
+import org.semanticweb.owlapi.model.OWLAxiom
+import org.semanticweb.owlapi.model.IRI
+
+object BgeeExpressionToOWL {
+
+  def strToSource(str: String): Source = {
+    io.Source.fromFile(str)
+  }
+
+  def convert(expressionData: Source): Set[OWLAxiom] = expressionData.getLines.flatMap(translate).toSet[OWLAxiom]
+
+  def translate(expressionLine: String): Set[OWLAxiom] = {
+    val items = expressionLine.split("\t", -1)
+    if (items(6).startsWith("absent") || items(6).startsWith("Expression")) {
+      Set.empty
+    } else {
+      val axioms = mutable.Set.empty[OWLAxiom]
+      val geneID = StringUtils.stripToNull(items(0))
+      val anatomicalID = StringUtils.stripToNull(items(2))
+
+      val expression = OntUtil.nextIndividual()
+      axioms.add(Declaration(expression))
+      axioms.add(expression Type GeneExpression)
+
+      val structure = OntUtil.nextIndividual()
+      axioms.add(Declaration(structure))
+
+      val term = (Option(StringUtils.stripToNull(items(2))).filter(_ != "\\").get)
```

Comment:
Oh, I see that I myself introduced this pattern in `ZFINExpressionToOWL`... :-D

ZFIN started putting backslashes in place of empty values and I wanted to make sure they were treated as empty (and I am okay with failing with NullPointerException because those fields are not supposed to be empty). So mostly ignore the previous comment, but perhaps there is no need for checking for backslashes with these data files? And no need to put the Option in.